### DISCUSSION TITLE
Where are interesting Cura Forks like this one collected?

### DISCUSSION THREAD

**Discussion by DerGenaue** [smartavionics/Cura#144] [General]
Is there some place where interesting Cura Forks are collected?
I only found out about this one because someone linked it on Reddit.

There is eg. another Cura Fork where a friend of mine replaced the Tree Support implementation:

https://www.reddit.com/r/3Dprinting/comments/p7n4x4/i_created_a_new_tree_support_implementation_for/
https://github.com/ThomasRahm/CuraEngine

**Comment by smartavionics** [comment]
Sorry, I'm not aware of any collection of Cura forks. I would have thought that if they are public then they would be here on github?

**Reply by DerGenaue** [reply]
Yeah; I mean you can search through the hundreds of Cura forks; but that doesn't give you a list of the ones that are actually interesting and contain new features / improvements
